chronos_bin/src/message_processor.rs

```diff
@@ -112,20 +131,14 @@ impl MessageProcessor {
                     Err(e) => {
                         if e.contains("could not serialize access due to concurrent update") && retry_count < max_retry_count {
                             //retry goes here
-                            eprintln!("retrying");
+                            log::info!("retrying");
```

Comment:
When this will be sent to log collector, it might not be very useful. Especially in the multi-threading context the log like without much context may appear as random noise.